Yoshi/.swiftlint.yml

```diff
@@ -2,4 +2,3 @@ line_length: 120
 excluded:
 - Pods
 disabled_rules:
-- force_cast
```

Comment:
I think you should disable this menu per file as you need it instead of disabling it for the whole project.

Reply:
SwiftLint is removed